kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/bluetooth/BluetoothGatt.java

```diff
@@ -29,7 +29,7 @@
 	 * 
 	 * @return If connection was successful
 	 */
-	public boolean connect() throws KuraException;
+	public boolean connect(String adapterName) throws KuraException;
```

Comment:
@pierantoniomerlino This is API code: we cannot change it in this way. I suggest you to overload the connect method adding one with the adapterName parameter, but we cannot remove the old method. You need, also, to add a `@since` in the comment to specify that this API is available since org.eclipse.kura.bluetooth;version="1.2.0" like @cdealti did here: https://github.com/eclipse/kura/blob/develop/kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/configuration/ConfigurationService.java#L85